docs/tables/aws_route53_zone.md

```diff
@@ -58,3 +56,32 @@ from
 where
   name like '%.turbot.com.'
 ```
+
+### List VPCs associated with zones
+```sql
+select 
+  name,
+  id,
+  v ->> 'VPCId'as vpc_id,
```

Comment:
@ParthaI Can you please make sure queries are formatted properly, e.g,`v ->> 'VPCId' as vpc_id`